flathub.json

```diff
@@ -0,0 +1,4 @@
+{
+  "publish-delay-hours": 1,
```

Comment:
Please keep it at 3. It makes sense to set it to 0 or 1 for extra-data apps but otherwise default is preferred.